conformance/tests/ot_has.py

```diff
@@ -195,21 +195,42 @@ def test_has_gt(man):
     return errors
 
 
+def test_has_gt_nil(man):
+    """None, nil translates to 0 when refering to numeric values,
+    so this evaluates to return All characters with height value greater than 0"""
+    errors = []
+    G = man.setGraph("swapi")
+    count = 0
+    for i in G.query().V().has(gripql.gt("height", None)):
```

Comment:
I don't think this is the behavior we want. See https://learnsql.com/blog/null-comparison-operators/

Another example would be:
```
G.query().V().has(gripql.eq("height", None))
```
I don't think it should find height == 0, it should find instances of height === nil